src/App.js

```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './component/Calculator';
 
 function App() {
```

Comment:
- [ ] Kindly make the `App` component a class-based component based on project requirements. 👍